tests/integration/priority.js

```diff
@@ -0,0 +1,152 @@
+import expect from 'expect.js'
```

Comment:
@kof No problem, easy to move. I set my tests up a little differently and generally do unit+integration and toss these in with integration. Sorry for the mixup!